### ISSUE TITLE
Unable to perform make preprocess_data for rnnt benchmark

### ISSUE THREAD

**Issue by anrahman4** [mlcommons/inference_results_v2.0#2]
Hello. I am currently trying to run the MLCommons provided rnnt benchmark to run inference tests. However, I was unable to successfully perform the make preprocess_data for the rnnt benchmark inside the running container. I am currently running it on a Dell PowerEdge R7525 that has one NVIDIA A40 GPU connected into one of the riser slots as a PCIe card. 

**_I get the following:_** 
[error.txt](https://github.com/mlcommons/inference_results_v2.0/files/8477700/error.txt)

```
root@mlperf-inference-root-x86_64:/work# make preprocess_data BENCHMARKS=rnnt
[2022-04-13 01:52:06,300 preprocess_data.py:52 INFO] Updating reference repo for convert_librispeech.py script...
[2022-04-13 01:52:06,300 __init__.py:43 INFO] Running command: make clone_loadgen
make[1]: Entering directory '/work'
Updating Loadgen
HEAD is now at 68c681b Fix submission checker: power server script prints UTC timestamps since v1.1
make[1]: Leaving directory '/work'
[2022-04-13 01:52:14,603 preprocess_data.py:59 INFO] Converting flac -> wav and generating manifest.json for test set...
[2022-04-13 01:52:14,603 __init__.py:43 INFO] Running command: cd build/inference/speech_recognition/rnnt && python3 pytorch/utils/convert_librispeech.py --input_dir /data/LibriSpeech/dev-clean --dest_dir /preprocessed_data/LibriSpeech/dev-clean-wav --output_json /preprocessed_data/LibriSpeech/dev-clean-wav.json
[/preprocessed_data/LibriSpeech/dev-clean-wav.json] Scaning input dir...
[/preprocessed_data/LibriSpeech/dev-clean-wav.json] Converting audio files...
0it [00:00, ?it/s]
[/preprocessed_data/LibriSpeech/dev-clean-wav.json] Generating json...
Traceback (most recent call last):
  File "pytorch/utils/convert_librispeech.py", line 81, in <module>
    with open(args.output_json, 'w') as fp:
FileNotFoundError: [Errno 2] No such file or directory: '/preprocessed_data/LibriSpeech/dev-clean-wav.json'
Traceback (most recent call last):
  File "code/rnnt/tensorrt/preprocess_data.py", line 192, in <module>
    main()
  File "code/rnnt/tensorrt/preprocess_data.py", line 186, in main
    preprocess_rnnt(data_dir, preprocessed_data_dir)
  File "code/rnnt/tensorrt/preprocess_data.py", line 60, in preprocess_rnnt
    flac_to_wav(absolute_data_dir, librispeech_path, "dev-clean", "dev-clean-wav")
  File "code/rnnt/tensorrt/preprocess_data.py", line 47, in flac_to_wav
    run_command(script_cmd)
  File "/work/code/common/__init__.py", line 45, in run_command
    return subprocess.check_call(cmd, shell=True)
  File "/usr/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'cd build/inference/speech_recognition/rnnt && python3 pytorch/utils/convert_librispeech.py --input_dir /data/LibriSpeech/dev-clean --dest_dir /preprocessed_data/LibriSpeech/dev-clean-wav --output_json /preprocessed_data/LibriSpeech/dev-clean-wav.json' returned non-zero exit status 1.
make: *** [Makefile:467: preprocess_data] Error 1
```

**_Here are the steps I followed to reproduce the issue:_**

1.) Install Ubuntu 20.04.4
2.) Install NVIDIA Driver 510.47.03
`sudo nvidia-smi --ecc-config=1`

#############################
Setup docker 
#############################

```
curl https://get.docker.com | sh \
  && sudo systemctl --now enable docker
```

#############################
Setup nvidia-docker runtime
#############################

```
distribution=$(. /etc/os-release;echo $ID$VERSION_ID) \
      && curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg \
      && curl -s -L https://nvidia.github.io/libnvidia-container/$distribution/libnvidia-container.list | \
            sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | \
            sudo tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
```
			
```
curl -s -L https://nvidia.github.io/libnvidia-container/experimental/$distribution/libnvidia-container.list | \
    sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | \
    sudo tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
```
	
```
sudo apt-get update
sudo apt-get install -y nvidia-docker2
sudo systemctl restart docker
```

#############################
Test working config
#############################

`sudo docker run --rm --gpus all nvidia/cuda:11.0-base nvidia-smi`

###############################################
Create mountpoint 
###############################################

```
sudo mdadm -C /dev/md0 --assume-clean -l raid10 -p n2 -c 512K  -n 4 /dev/nvme1n1 /dev/nvme2n1 /dev/nvme3n1 /dev/nvme4n1 
sudo mkfs.xfs /dev/md0 
sudo mount /dev/md0 /mnt/
```

#############################
Install MLPerf
#############################

```
cd /mnt/ 
sudo git clone https://github.com/mlcommons/inference_results_v2.0.git
cd inference_results_v2.0/closed/DellEMC
mkdir scratch
export MLPERF_SCRATCH_PATH=/mnt/inference_results_v2.0/closed/Dell/scratch
```

#############################
Build the MLPerf docker image
#############################

```
cd /mnt/inference_results_v2.0/
sudo make prebuild 
```

Inside running container environment:
`make build`

#############################
download data for benchmarks
#############################

```
make download_model BENCHMARKS=rnnt
make download_data BENCHMARKS=rnnt 
make preprocess_data BENCHMARKS=rnnt
```

**_Here is the nvidia-smi output for the system:_**

```
root@mlperf-inference-root-x86_64:/work# nvidia-smi
Wed Apr 13 02:20:40 2022
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 510.47.03    Driver Version: 510.47.03    CUDA Version: 11.6     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  NVIDIA A40          Off  | 00000000:81:00.0 Off |                    0 |
|  0%   37C    P0    73W / 300W |      0MiB / 46068MiB |      2%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
```

I am not sure why this is occurring:

FileNotFoundError: [Errno 2] No such file or directory: '/preprocessed_data/LibriSpeech/dev-clean-wav.json' 

Please let me know if you need additional information to help assist in troubleshooting this issue.

**Comment by anrahman4** [comment]
I have realized that when you run the commands below, they automatically get downloaded to the /home/ directory. 

```
make download_model BENCHMARKS=rnnt
make download_data BENCHMARKS=rnnt
```

Once I moved the LibriSpeech folder over to /preprocessed_data directory, it was able to preprocess the data properly. 

Are there any specific export commands that need to be run outside of the container or inside in advance such that the data is downloaded to the proper directories?
